src/CMakeLists.txt

```diff
@@ -356,9 +356,24 @@ if(ENABLE_GUI)
             ${FONTCONFIG_LIBRARIES})
     endif()
 
-    if(MSVC)
-        set_target_properties(solvespace PROPERTIES
-            LINK_FLAGS "/MANIFEST:NO /SAFESEH:NO /INCREMENTAL:NO /OPT:REF")
```

Comment:
Ugh, I'm only now noticing that I used `else()` instead of `elseif()`. This doesn't actually break anything, because MINGW simply doesn't have that flag set, so it would be a no-op. But this should really be fixed to use `elseif()` instead at some point, at least for clarity.